packages/tpetra/core/ext/TpetraExt_TripleMatrixMultiply_def.hpp

```diff
@@ -636,10 +765,10 @@ namespace Tpetra {
       // each row.  Chris Siefert says that this reflects experience in
       // ML; for the non-threaded case, ML found it faster to spend less
       // effort on estimation and risk an occasional reallocation.
-      size_t nnzAllocated = std::max(Ac_estimate_nnz(*Aview.origMatrix, *Pview.origMatrix), n);
-      Acrowptr_RCP.resize(m+1);          Acrowptr = Acrowptr_RCP();
-      Accolind_RCP.resize(nnzAllocated); Accolind = Accolind_RCP();
-      Acvals_RCP.resize(nnzAllocated);   Acvals   = Acvals_RCP();
+      size_t CSR_alloc = std::max(C_estimate_nnz(*Aview.origMatrix, *Pview.origMatrix), n);
+      lno_view_t Crowptr("Crowptr",m+1);
```

Comment:
These View allocations initialize.  The previous ArrayRCP allocations did not.  If you don't want to zero initialize, do this:
```c++
lno_view_type Crowptr(Kokkos::view_alloc ("Crowptr", Kokkos::WithoutInitializing), m+1);
```

Reply:
🤦‍♂️